docs/adr/0003-configuration.md

```diff
@@ -0,0 +1,19 @@
+# 3. Configuration
```

Comment:
Would you mind making this more specific?  Like `Configuration by Environment Variables`?